source/options.css

```diff
@@ -31,13 +22,13 @@ h3,
 h4 {
 	width: 100%;
 	margin-top: 0;
-	margin-bottom: 0.25em;
+	margin-bottom: 0.25rem;
```

Comment:
Unless you're going for very specific spacing, you could drop `margin-bottom` because all of these elements already have it

Reply:
Their default top and bottom margin of `1.33em` is too much.